osd/infranode_resized_auto.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "_tags": ["t_config"],
+    "service_name": "SREManualAction",
+    "summary": "Infra nodes resized",
+    "description": "SRE has resized the infra nodes of your cluster to ${INSTANCE_TYPE} to accommodate cluster load. For more information, please visit https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#node-scaling-after-installation_rosa-limits-scalability .",
```

Comment:
```suggestion
    "description": "SRE has resized the infra nodes of your cluster to ${INSTANCE_TYPE} to accommodate cluster load. For more information, please visit https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#node-scaling-after-installation_rosa-limits-scalability.",
```

Reply:
Won't the trailing . be pulled into URL detection without a space separating it? I don't think we'd want customers to click on the link and get a 404

Reply:
Hm, good point - I didn't think about that. I was just looking at it like a typo haha.

It does seem like going to `https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#node-scaling-after-installation_rosa-limits-scalability.` does the right thing and doesn't 404 though. Though if we want to guard against this, I think we can word-smith it a bit like:

"...please visit ${URL} to learn more." to avoid the space before the period